cluster/proto/cluster/message.proto

```diff
@@ -2,48 +2,40 @@ syntax = "proto3";
 
 package weaviate.cloud.internal.cluster;
 
-//option go_package = "github.com/weaviate/weaviate/cloud/proto/cluster";
+service ClusterService {
```

Comment:
Just re-ordered the file to have the service at the top and then the subsequent Request/Response ordered one after the other.